scripts/commons.ts

```diff
@@ -0,0 +1,47 @@
+import { join } from 'https://deno.land/std@0.224.0/path/mod.ts'
+
+export const backupFile = (filePath: string) =>
+  Deno.copyFile(filePath, `${filePath}.bak`)
+
+export const insertLineAfterString = (
+  multiLineText: string,
+  searchString: string,
+  lineToInsert: string,
+  options?: { insertBefore?: boolean }
+): string => {
+  const lines = multiLineText.split('\n')
+
+  let insertIndex = -1
+
+  for (let i = 0; i < lines.length; i++) {
+    if (lines[i].includes(searchString)) {
+      insertIndex = i
+      break
+    }
+  }
+
+  if (insertIndex !== -1) {
+    const insertPosition = options?.insertBefore ? insertIndex : insertIndex + 1
+    lines.splice(insertPosition, 0, lineToInsert)
+  }
+
+  return lines.join('\n')
+}
+
+export const getMapsApiKey = async (projectPath: string) => {
+  try {
+    const protonBundlePath = join(projectPath, 'proton-bundle.json')
+    const protonBundleText = await Deno.readTextFile(protonBundlePath)
+    const protonBundle = JSON.parse(protonBundleText)
```

Comment:
Can this be done in one go?
`const module = await import(protonBundlePath, {
  with: { type: "json" },
});`

Reply:
Yuuuup